queries.sql

```diff
@@ -9,6 +9,25 @@ SELECT * FROM animals WHERE neutered=TRUE;
 SELECT * FROM animals WHERE name!='Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+/* Inside a transaction update the animals table by setting the species column to unspecified. Then roll back  */
+BEGIN;
+UPDATE animals
+SET species = 'Unspecified';
+ROLLBACK;
```

Comment:
Hello,

Great Job adding the transaction as suggested, It’s clear that you’ve understood the requirement to perform an update within a transaction and to roll it back afterward. However, there are a few points that need to be addressed to meet the project's specific requirements fully:

1. **Verification Step Missing:**
   The project requirements explicitly ask to verify that the change has been made before rolling it back. This means you should include a SELECT statement after the UPDATE and before the ROLLBACK to confirm that the `species` column has been updated to 'Unspecified'.

Here’s how you could modify your code to include the verification step:

```sql
BEGIN; -- Starts a transaction (verify this command with your SQL database dialect)

-- Update the species to 'Unspecified'
UPDATE animals
SET species = 'Unspecified';

-- Select statement to verify the update
SELECT * FROM animals WHERE species = 'Unspecified';

-- Rollback the transaction to undo the update
ROLLBACK;
```

Keep up the good work!

---